app/AppKernel.php

```diff
@@ -37,11 +37,15 @@ public function registerBundles()
             new FOS\RestBundle\FOSRestBundle(),
             new JMS\SerializerBundle\JMSSerializerBundle(),
 
+            // Application bundles
+            new AppBundle\AppBundle(),
+
             // Remove this Bundle when using the SE as the basis for a new project
             new Acme\DemoBundle\AcmeDemoBundle(),
```

Comment:
really? I do not see a DemoBundle here https://github.com/symfony/symfony-standard/blob/2.7/app/AppKernel.php